src/actions/vstudio/vs2005_csproj_user.lua

```diff
@@ -57,6 +57,7 @@
 				if #contents[cfg] > 0 then
 					p.push('<PropertyGroup %s>', m.condition(cfg))
 					p.outln(contents[cfg])
+                    m.localDebuggerWorkingDirectory(cfg)
```

Comment:
This should be put inside the call array `m.elements.userConfigPropertyGroup` defined on line 21 of this file.